### ISSUE TITLE
The trans twig filter generates non translatable code

### ISSUE THREAD

**Issue by nijel** [phpmyadmin/phpmyadmin#13794]
The trans filter generates code which gettext fails to extract and thus such strings are no longer translatable.

I've noticed it on following code, but it might happen in more cases:

https://github.com/phpmyadmin/phpmyadmin/blob/5acd64f84d964e5f2db6170ecb5a42581e74ca12/templates/columns_definitions/table_fields_definitions.twig#L27-L30

This generates:

```php
        echo PhpMyAdmin\Util::showHint(_gettext(("For default values, please enter just a single value, without" . " backslash escaping or quotes, using this format: a")));
```

The problem here is string concatenation as xgettext doesn't deal with that.

**Comment by nijel** [comment]
I've workarounded this by removing concatenation in the templates (see 2ed01429503e2019870044d9fcb646c15af37740), but maybe we should look for better fix.

**Comment by mauriciofauth** [comment]
The problem here is not the string concatenation but the extra parentheses around the string.

**Comment by nijel** [comment]
You're right, the parenthesis cause the problem, concatenation works fine.

**Comment by mauriciofauth** [comment]
This has nothing to do with Twig. I think it's a bug in _xgettext_.

For example, _xgettext_ fails to extract the string from this:
```php
echo gettext(('text'));
```

**Comment by mauriciofauth** [comment]
The easiest solution is to avoid complex strings in the `trans` filter.

Other solutions would be to extend Twig so that it does not generate the parentheses or write a script that removes the parentheses. However, I don't think these are good solutions.
